config/locales/en.yml

```diff
@@ -1347,11 +1347,11 @@ en:
         start_date_verification_form:
           attributes:
             trainee_has_started_course:
-              blank: Choose yes or no
+              blank: Select yes or no
```

Comment:
@gregknight1 — i think this should be more verbose?

Reply:
hmm I carded this up here: https://trello.com/c/b4vA5njE/3269-snag-did-the-trainee-start-their-itt-error-content

Reply:
ahh yes, and it's been picked up in this open PR: https://github.com/DFE-Digital/register-trainee-teachers/pull/1737/files?diff=unified&w=0#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R1354

@jhenderson / @archferns - beware of the merge!

Reply:
@ralph-hawkins @jhenderson Should be - **Select yes if the trainee started the course**

Reply:
@gregknight1 I've update line 1269.

Reply:
@AbigailMcP ah nice — @jhenderson sorry, i didn't know this was in another PR